source/atlas-cli-changelog.txt

```diff
@@ -14,6 +14,48 @@
    :depth: 1
    :class: singlecol
 
+.. _atlas-cli-1.12.0:
+
+{+atlas-cli+} 1.12.0
+--------------------
+
+- Drops support for Ubuntu 18.04.
+- Adds :ref:`atlas-deployments` commands so that you can work with
+  |service|, including using |fts| and {+avs+}, throughout the entire 
+  software development lifecycle, from your local environment to the 
+  cloud. To learn more, see :ref:`atlas-cli-local-cloud`.
+- Adds the following new commands to manage {+atlas-sp+} instances
+  and connection registries:
```

Comment:
Question: are these the first streams processing commands or were there other ones released before this? If they're the first consider just linking to the top-level streams processing command to align with what you did with atlas deployments (or, conversely, list out the deployments commands). This is optional though, not blocking